services/offset-earth/offset-earth-carbon.tester.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+const { withRegex } = require('../test-validators')
+
+t.create('request for existing profile')
+  .get('/offsetearth.json')
+  .expectBadge({
+    label: 'carbon offset',
+    message: withRegex(/[\d.]+ tonnes/),
+  })
+
+t.create('request for existing profile (mock)')
```

Comment:
Do we need this test considering the live one above? If so what are we using this test to validate above and beyond the first one?

Our test runner also automatically flags each test as live or mock in the output so the trailing `(mock)` would be a bit duplicative

Reply:
I'll remove this test, and remove the `(mock)` text (it was just taken from another example service). 👍

Reply:
No worries, thanks for updating!